src/ZEO/ClientStorage.py

```diff
@@ -222,7 +222,8 @@ def __init__(self, addr, storage='1', cache_size=20 * MB,
         username
         password
         realm
-            Credentials for authentication to server.
+            [ZEO4 only] Credentials for authentication to server.
+            In ZEO5 support for credentials has been dropped in favor or SSL.
```

Comment:
```suggestion
            In ZEO5 support for credentials has been dropped in favor of SSL.
```

Reply:
@jugmac00, thanks for the catch. I'm incorporating your change in my next update.